src/App.test.js

```diff
@@ -10,15 +15,11 @@ const messages = {
   en: messages_en,
 };
 
-const language = 'en';
-
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(
-    <IntlProvider locale={language} messages={messages[language]}>
+it('renders welcome message', () => {
+  const { getByText } = render(
```

Comment:
Sorry, I am not very familiar with ES6 notation, could you explain me in English or Spanish words what is the meaning of this line?